src/modules/esl-base-element/README.md

```diff
@@ -1,28 +1,57 @@
 # [ESL](../../../) Base Element
 
-Version: *1.0.0*
+Version: *1.1.0*
 
 Authors: *Alexey Stsefanovich (ala'n)*
 
 <a name="intro"></a>
 
-Provides the ESLBaseElement - a base class for custom element declaration, and a set of common decorators.
+Provides the core to create custom elements and ts decorators to simplify components creation.
 
-### Available decorators:
- - `@attr` - to map string type property to HTML attribute.
- - `@boolAttr` - to map boolean property to HTML boolean (marker) attribute state.
- - `@jsonAttr` - to map object property to HTML attribute using JSON format to serialize / deserialize value.
-
-Use `@override` or `@constant` decorator to override property that was created 
-via `@attr`, `@boolAttr` or `@jsonAttr` on the parent level.
+## Base elements:
+- **ESLBaseElement** - base class for standard custom element declaration
+- **ESLMixinElement** (beta) - base class to create mixin element that attaches to element via custom attribute
 
 ### Base Element static API
 - `MyElement.is` - property that defines tag name
-  
-- `MyElement.register` - calls registration inside customElements registry
+- `MyElement.observedAttributes` - array of attributes to observe
+
+- `MyElement.register` - register component inside `customElements` registry
 - `MyElement.registered` - returns promise that will be resolved as soon as the component is registered
 
-### Example
+### Mixin Element static API (beta)
+- `MyMixinElement.is` - property that defines connection attribute name
+- `MyMixinElement.observedAttributes` - array of additional attributes to observe
+
+- `MyMixinElement.register` - register component inside `ESLMixinRegistry`
+
+### Shared Element API
+- `connectedCallback` - called when element appends to the DOM
+- `disconnectedCallback` - called when element is disconnected from the DOM
+- `attributeChangeCallback` - called when observed attribute has changed
+
+- `$$cls` - check or change element CSS classes (uses CSSClassUtils) 
+- `$$attr` - check or change element attributes
+- `$$fire` - dispatch event with `esl:` prefix
+
+### Base Element API
+- `connected` - readonly marker, true if element connected and base `connectedCallback` executed
+
+### Mixin Element API (beta)
+- `$host` - readonly mixin target DOM element
+
+### Element decorators:
+Works for both `ESLBaseElement` and `ESLMixinElement`.
+
+ - `@attr` - to map string type property to HTML attribute.
```

Comment:
As I remember we don't use a dot in such type of phrase (2 following phrases also)